.travis.yml

```diff
@@ -1,18 +1,17 @@
 language: clojure
-lein: lein2
 jdk:
+  - oraclejdk9
   - oraclejdk8
-  - oraclejdk7
-  - openjdk7
-script: lein2 test :all
+script: lein test :all
 before_script:
-  - psql -c 'create database jdbc_util_test;' -U postgres
+  - psql -c "create user jdbc_util_test with createdb createrole password 'foobar'" -U postgres
+  - psql -c 'create database jdbc_util_test owner jdbc_util_test' -U postgres
 env:
   global:
     - JDBCUTIL_DBNAME=//127.0.0.1:5432/jdbc_util_test
-    - JDBCUTIL_DBUSER=postgres
```

Comment:
These variables seem like they should work, from a quick look at the test setup.

Reply:
there's a test that is expecting the test user to not be a superuser